playbook/appherd/920_django_fix_permissions.yml

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
To clarify, this playbook only needs to be run once per environment, right?

Reply:
All of these changes need to be deployed. the fix_permissions is actually adding the management command to the codebase. I have added a PR for the ansible scripts that would allow the management command to be run once in an environment. The same applies for the cron exports. These are the management commands to be added. Using them is done via the deployment/jenkins platform.